src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -240,13 +338,37 @@ static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro)
     }
 #endif
 
-#ifdef  USE_GYRO_SPI_MPU9250
-#ifndef USE_DUAL_GYRO
-    spiBusSetInstance(&gyro->bus, MPU9250_SPI_INSTANCE);
+#ifdef USE_GYRO_IMUF9001
+    #ifdef IMUF9001_SPI_INSTANCE
+        spiBusSetInstance(&gyro->bus, IMUF9001_SPI_INSTANCE);
+    #else
+        #error IMUF9001 is SPI only
+    #endif
+    #ifdef IMUF9001_CS_PIN
```

Comment:
These defines can't be in here, as they will break when universal targets are introduced. Please rewrite to support configuration at runtime.